CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 # Change Log
 
+## [0.20.0](https://github.com/eclipse/lemminx/milestone/32?closed=1) (March 29, 2022)
+
+### Enhancements
+
+ * Fold closing tags too. See [#1178](https://github.com/eclipse/lemminx/issues/1178).
```

Comment:
Closing tags should be included in the code folding range.

Reply:
fixed.